server/routes/linktaFlowRouter.ts

```diff
@@ -1,20 +1,45 @@
 import { Router } from 'express';
-import {
-  fetchLinktaFlows,
-  fetchLinktaFlow,
-  updateLinktaFlow,
-  deleteLinktaFlow,
-} from '@controllers/linktaFlowController';
+import type { Request, Response } from 'express';
+import createLinktaFlowController from '@/controllers/linktaFlowController';
+import createLinktaFlowService from '@/services/linktaFlowService';
 import isAuthorized from '@middleware/firebaseAuthMiddleware';
 
 const linktaFlowRouter = Router();
 
-linktaFlowRouter.get('/', isAuthorized, fetchLinktaFlows);
+linktaFlowRouter.use(isAuthorized);
```

Comment:
@kinskiwu This is fine and I'll approve it, but just FYI: My thinking was we would implement the auth middleware per route so that it could be omitted from some routes if desired. I don't know if/when we'll need to do that.

Reply:
I was thinking to apply auth to all linktaflow routes for now and if we have a new route which will not need auth in the future then we will make the switch. With this implementation when we add new routes that need auth we don't need to manually add the auth middleware. Guess I was just lazy lol